components/locale-provider/index.tsx

```diff
@@ -53,7 +53,7 @@ export default class LocaleProvider extends React.Component<LocaleProviderProps,
     };
   }
 
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
It breaks ci, fixed in https://github.com/ant-design/ant-design/commit/e3446840face7224a7534f0ed9c506ecf3de4198